Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasket-AddNonprofitsViewController.m

```diff
@@ -71,9 +72,19 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
     Nonprofit *nonprofit = self.allNonprofits[indexPath.row];
     //TURN COLOR CHANGE OR SELECTION UI
+    NonprofitListCell *cell = [tableView cellForRowAtIndexPath:indexPath];
+    cell.backgroundColor = [UIColor blueColor];
+    
     [self.basket.nonprofits addObject:nonprofit];
 }
 
+- (void)tableView:(UITableView *)tableView didDeselectRowAtIndexPath:(NSIndexPath *)indexPath {
+    Nonprofit *nonprofit = self.allNonprofits[indexPath.row];
+    NonprofitListCell *cell = [tableView cellForRowAtIndexPath:indexPath];
+    cell.backgroundColor = [UIColor whiteColor];
```

Comment:
Same here, you can use the `allowsMultipleSelection` property of the `UITableView` to allow multiple selection without any custom logic.